command/genesis/polybft_params.go

```diff
@@ -606,6 +606,15 @@ func (p *genesisParams) getValidatorAccounts() ([]*validator.GenesisValidator, e
 				}
 			}
 
+			if p.stakeTokenAddr != contracts.NativeERC20TokenContract {
+				s, exists := p.stakeInfos[addr]
+				if !exists {
+					stake = command.DefaultStake
+				} else {
+					stake = s
+				}
+			}
```

Comment:
Merge this part with the previous check, because the logic for `stakeInfos` population is the same:

```suggestion
			if p.nativeTokenConfig.IsMintable || p.stakeTokenAddr != contracts.NativeERC20TokenContract {
				s, exists := p.stakeInfos[addr]
				if !exists {
					stake = command.DefaultStake
				} else {
					stake = s
				}
			}
```